srcpkgs/bruno/template

```diff
@@ -0,0 +1,43 @@
+# Template file for 'bruno'
+pkgname=bruno
+version=2.1.0
+revision=1
+archs="x86_64"
+hostmakedepends="n python3 python3-distutils-extra"
+short_desc="Fast and Git-Friendly Opensource API client"
+maintainer="Orphaned <orphan@voidlinux.org>"
```

Comment:
I don't think that new orphaned packages will get accepted. If you aren't willing to mark yourself as the maintainer of the package, why are you contributing it?

Reply:
Hello @meator!
Thank you for your review.

I see the PR/this repo as a place to find some packages that are not officially supported. A bit like AUR.
And I thought it would be nice to share this template for people that want it, while I am not sure to be around on void for a long time neither.

I am not sure what the maintainer is for neither, I like the idea this is public.

So this is why I prefer not to put my name on it, but I understand your point.

Reply:
I've put myself in the maintainer